src/Shared/Components/SelectPicker/utils.ts

```diff
@@ -118,7 +118,7 @@ export const getCommonSelectStyle = <OptionValue, IsMulti extends boolean>({
     control: (base, state) => ({
         ...base,
         minHeight: size === ComponentSizeType.medium ? 'auto' : '36px',
-        minWidth: '100px',
+        minWidth: '56px',
```

Comment:
will it impact other areas

Reply:
Have checked the impactged areas as well